manifests/init.pp

```diff
@@ -113,50 +113,59 @@
 #  prometheus 1.8.*, only durations understood by golang's time.ParseDuration are supported. Starting
 #  with prometheus 2, durations can also be given in days, weeks and years.
 #
+#  [*collect_scrape_jobs*]
+#  Array of scrape_configs. Format, e.g.:
+#  - job_name: some_exporter
+#    scheme: https
+#  The jobs defined here will be used to collect resources exported via prometheus::daemon,
```

Comment:
i find this datastructure a little over-complicated and inconvenient - what if i want to collect all jobs, regardless of the type? the way it's setup now, i have to specify each one by hand....

i'm thinking of simplifying this and simply make that a mixed type of boolean / array of job names. if it's a list, it behaves basically the same way (minus the scheme override, which belongs in the scrape_job def). if it's a boolean and true, all jobs are collected. if it's false, nothing is collected of course.

i'm working on an update of this PR to make it work with latest master which includes those changes.

Reply:
Thanks for picking up the work!
I'm not sure I understand your suggestion though. I'll follow up on your new PR #304.